README.md

```diff
@@ -1,127 +1,126 @@
 Docker open-oni
 ===============
 
-Run open-oni (community chronicling america fork) in Docker.
+(Production instructions TBD - for now this is just for doing open-oni dev)
```

Comment:
I can link for sure, but that is a good question.  Not sure what we should do.  We don't want OSX, Windows, Linux, etc. all clogging up this file, I don't think.  But we REALLY don't want things getting out of sync.  As it is, the README is already a big duplication of dev.sh :-/